src/select-menu/src/OptionsList.js

```diff
@@ -139,11 +139,12 @@ const OptionsList = memo(function OptionsList(props) {
 
   const handleSelect = useCallback(
     item => {
-      if (isSelected(item) && isMultiSelect) {
-        onDeselect(item)
-      } else {
-        onSelect(item)
-      }
+      // THE ONDESELECT CONDITION REQUIRES THE USER TO CLICK TWICE TO RE-SELECT ON MULTIPLE
+      // if (isSelected(item) && isMultiSelect) {
+      //   onDeselect(item)
+      // } else {
+      onSelect(item)
+      // }
```

Comment:
I think we want to keep the conditional in place. It allows multiselects to deselect items by clicking on them again.

What may be the problem is that this callback isn't specifying all of its dependencies: `close` `onSelect` `isSelected`

Reply:
Although, it does look as though onDeselect is still being called under the hood by the underlying TableRow item.